pepper-apis/studybuilder-cli/studies/singular/i18n/en.conf

```
@@ -1623,19 +1623,19 @@
       "li3": "I may cancel this authorization at any time by submitting a written request to Project Singular over mail or email, except if Project Singular has already relied upon it (for example, once information is released, it will not be retrieved). "
       "li4": "My questions about this authorization form have been answered."
-      "li5": "I understand that access to my $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"child's\", \"\") medical records may include access to my $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"child's\", \"\") previous genetic testing results. "
-      "li6": "I have already read and signed the informed consent document for this study, which describes the use of my $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"child's\", \"\") personal health information, and hereby grant permission to Dr. Kirstie Keller, and/or members of the study staff team to examine copies of my $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"child's\", \"\") medical records pertaining to my $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"child's\", \"\") single ventricle defect. I acknowledge that a copy of this completed form will be available on my online dashboard at www.projectsingular.org."
+      "li5": "I understand that access to my $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"child's\", \"\") medical records may include access to $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"their\", \"my\") previous genetic testing results. "
+      "li6": "I have already read and signed the informed consent document for this study, which describes the use of my $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"child's\", \"\") personal health information, and hereby grant permission to Dr. Kirstie Keller, and/or members of the study staff team to examine copies of my $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"child's\", \"\") medical records pertaining to $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"their\", \"my\") single ventricle defect. I acknowledge that a copy of this completed form will be available on my online dashboard at www.projectsingular.org."
     }
     "s3": {
       "p1": "$ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"By completing this information, you are agreeing to allow us to contact these physician(s) and hospital(s)/institutions(s) to obtain your child’s records. If you decide to collect and upload your child’s medical records to Project Singular, you are giving permission to Project Singular study staff to obtain your records.\", \"C. By completing this information, you are agreeing to allow us to contact these physician(s) and hospital(s)/institutions(s) to obtain your records. If you decide to collect and send your medical records to Project Singular study staff you are giving permission to Project Singular study staff to obtain your records.\")",
```

I think there's a pre-existing typo here
```suggestion
    "s3": {
      "p1": "$ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"By completing this information, you are agreeing to allow us to contact these physician(s) and hospital(s)/institution(s) to obtain your child’s records. If you decide to collect and upload your child’s medical records to Project Singular, you are giving permission to Project Singular study staff to obtain your records.\", \"C. By completing this information, you are agreeing to allow us to contact these physician(s) and hospital(s)/institution(s) to obtain your records. If you decide to collect and send your medical records to Project Singular study staff you are giving permission to Project Singular study staff to obtain your records.\")",
```